modular_sand/code/modules/arousal/arousal.dm

```diff
@@ -0,0 +1,38 @@
+/mob/living/carbon/human/proc/pick_receiving_organ(mob/living/L)
+	if (!iscarbon(L))
+		return
+	var/mob/living/carbon/C = L
+	var/list/receivers_list
+	var/list/other_worn = C.get_equipped_items()
+	for(var/obj/item/organ/genital/G in C.internal_organs)
+		if((G.genital_flags & CAN_CUM_INTO) && G.is_exposed(other_worn)) //filter out what you can't cum into
+			LAZYADD(receivers_list, G)
+	if(LAZYLEN(receivers_list))
+		var/obj/item/organ/genital/ret_organ = input(src, "in what hole?", "Climax", null) as null|obj in receivers_list
+		return ret_organ
+
+/mob/living/proc/receive_climax(mob/living/partner, obj/item/organ/genital/receiver = null, obj/item/organ/genital/source_gen, spill = TRUE)
+	set_lust(0)
```

Comment:
So far only one of them did it, don't block the other one.

Reply:
wdym?

Reply:
`receive_climax()` (set_lust = 0) will prevent the receiving end from cumming, and so, only the guy fucking gets it.

Reply:
Also remove the send_signal under